### PR TITLE
Initial creation of the connector

### PR DESCRIPTION
**Pull request by diogopontual**

### CODE REVIEW

**Comment by diogopontual** [comment]
Hey @pinkforest. I think that the PR is good to go now! TY.

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+async-std = { version = "1", features = ["attributes", "tokio1"] }
+futures = { version = "0.3", default-features = false }
+serde = { version = "1.0", default-features = false, features = ["derive"]}
+serde_json = "1.0.122"
+anyhow = "1.0.86"
+elasticsearch = "8.5.0-alpha.1"
+async-trait = "0.1.81"
+
+fluvio = { git = "https://github.com/infinyon/fluvio", rev = "171fba00b4ae70094bfd54a4a32427baef9ecf02" }
+fluvio-connector-common = { git = "https://github.com/infinyon/fluvio", rev = "171fba00b4ae70094bfd54a4a32427baef9ecf02", features = ["derive"] }
```
Comment (digikata) [inline Cargo.toml:19]: This is sort of a bug from the cdk generate, but the rev should probably be pointed at a specific fluvio release (e.g. "v0.12.1").

**Review by digikata — APPROVED** [review]
Looks good to me,  minor nit the referenced fluvio rev. That could be fixed here or merged and fixed with a follow on PR.

**Comment by digikata** [comment]
Thanks @diogopontual! merged w/ minor cleanups and refresh after.

**Pull request merged**